index.bs

```diff
@@ -1919,47 +1919,39 @@ WebAuthn supports multiple attestation types:
     ECDAA-Issuer (see [[FIDOEcdaaAlgorithm]]).
 
 
-### Generating an Attestation Object ### {#generating-an-attestation-object}
+<h4 id="generating-an-attestation-object" algorithm>Generating an Attestation Object</h4>
 
-This section specifies the algorithm for generating an [=attestation object=] (see: [Figure 3](#fig-attStructs)) for any
-[=attestation statement format=].
+To generate an [=attestation object=] (see: [Figure 3](#fig-attStructs)) given:
 
-In order to construct an [=attestation object=] for a given [=public key credential=] using a particular [=attestation statement
-format=], the [=authenticator=] MUST first generate the [=authenticator data=].
+: |attestationFormat|
+:: An [=attestation statement format=]
+: |authData|
+:: A byte array containing [=authenticator data=].
+: |hash|
+:: A cryptographic hash expected to contain the [=hash of the serialized client data=].
```

Comment:
s/expected to contain/containing/ ?

perhaps just: 
:   |hash|
::  The [=hash of the serialized client data=].

..?

Reply:
Sure.